testsuite/features/build_validation/retail/proxy_container_branch_network.feature

```diff
@@ -0,0 +1,74 @@
+# Copyright (c) 2024 SUSE LLC
+# Licensed under the terms of the MIT license.
+
+@proxy
+@private_net
+@containerized_server
+Feature: Prepare the containerized branch server for PXE booting
+  In order to use SUSE Manager for Retail solution in a containerized setup
+  As the system administrator
+  I prepare the branch network in containerized setup
+
+  Scenario: Activate the branch network on the proxy
+    When I activate the private network on the proxy
+
+  Scenario: The DHCP server works as expected
+    Then the "dhcp_dns" host should be present on private network
+    And name resolution should work on private network
+
+  Scenario: Show the overview page of the containerized proxy
+    Given I am authorized for the "Admin" section
+    And I am on the Systems overview page of this "proxy"
+
+  Scenario: Create branch terminals group
+    When I follow the left menu "Systems > System Groups"
+    And I follow "Create Group"
+    And I enter "example" as "name"
+    And I enter "Terminal branch: example.org" as "description"
+    And I click on "Create Group"
+    Then I should see a "System group example created." text
+    # TODO: Re-add this when not using "Foreign" system type for the proxy anymore:
+    # When I follow "Target Systems"
+    # And I check the "proxy" client
+    # And I click on "Add Systems"
+    # Then I should see a "1 systems were added to example server group." text
+
+  Scenario: Create all terminals group
```

Comment:
As we discussed in other occasions I don't see a big deal of duplicated scenario titles, but be aware we have triplicated some of them in this PR ;)

Reply:
Right :cry: .

Since they have different tags, even if I use cucumber -n "name of scenario" only one will be executed.

But you are right. I'll see if I can make it a bit better before I merge.

Reply:
Not really the time. Left unchanged, to be done after 5.0.